app/services/foreman_ansible/proxy_selector.rb

```diff
@@ -0,0 +1,21 @@
+module ForemanAnsible
+  # Contains proxy selection rules for a host playbook run
+  class ProxySelector < ::ForemanTasks::ProxySelector
+    # rubocop:disable Metrics/MethodLength
+    def available_proxies(host)
+      proxies = {}
+
+      proxies[:fallback] = host.smart_proxies.with_features('Ansible')
+      proxy_scope = if Taxonomy.enabled_taxonomies.any?
+                      ::SmartProxy.with_taxonomy_scope_override(host.location,
```

Comment:
This doesn't work. I have a host with:

Location: 'Czech Republic' (no proxies)
Organization: 'E-Corp' (no proxies)

But this method still returns proxies in other organizations/locations? I'll investigate as I think it's a problem in core.